test_runner/performance/test_gc_feedback.py

```diff
@@ -79,3 +81,8 @@ def test_gc_feedback(neon_env_builder: NeonEnvBuilder, zenbenchmark: NeonBenchma
     zenbenchmark.record(
         "physical/logical ratio", physical_size / logical_size, "", MetricReport.LOWER_IS_BETTER
     )
+
+    layer_map_path = env.pageserver.timeline_dir(tenant_id, timeline_id) / "layer-map.json"
+    log.info(f"Writing layer map to {layer_map_path}")
+    with layer_map_path.open("w") as f:
```

Comment:
Does this file get preserved in practice?  I would have guessed you'd need to add it to `SMALL_DB_FILE_NAME_REGEX` to prevent it getting cleaned up.

Reply:
Usually I run with `--preserve-database-files`, but to try it out, I ran it once without and the file was still there. It makes sense: the `SMALL_DB_FILE_NAME_REGEX` keeps every file with a `.json` ending.

It might be confusing to the pageserver though to have foreign files in its directory, I think I'll change the path.